Source/Lib/Codec/EbEncHandle.c

```diff
@@ -3002,13 +3059,135 @@ EB_API EB_ERRORTYPE EbH265EncEosNal(
         outputStreamBuffer->pBuffer,
         (EB_U32*) &(outputStreamBuffer->nFilledLen),
         (EB_U32*) &(outputStreamBuffer->nAllocLen),
-        encodeContextPtr);
+        encodeContextPtr,
+        9999);
     
     *outputStreamPtr = outputStreamBuffer;
 
     return return_error;
 }
 
+/* charSet = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/" */
+static void baseDecodeFunction(EB_U8* encodedString, EB_U32 base64EncodeLength, EB_U8* decodedString )
+{
+    EB_U32 i, j, k = 0;
+    EB_U32 bitstream = 0;
+    EB_U32 countBits = 0;
+
+    // selects 4 characters from encodedString at a time
+    for (i = 0; i < base64EncodeLength; i += 4) {
+        bitstream = 0, countBits = 0;
+        for (j = 0; j < 4; j++) {
+            // make space for 6 bits
+            if (encodedString[i + j] != '=') {
+                bitstream = bitstream << 6;
+                countBits += 6;
+            }
+            // Finding the position of each encoded character in charSet and storing in bitstream
+            if (encodedString[i + j] >= 'A' && encodedString[i + j] <= 'Z')
+                bitstream = bitstream | (encodedString[i + j] - 'A');
+
+            else if (encodedString[i + j] >= 'a' && encodedString[i + j] <= 'z')
+                bitstream = bitstream | (encodedString[i + j] - 'a' + 26);
+
+            else if (encodedString[i + j] >= '0' && encodedString[i + j] <= '9')
+                bitstream = bitstream | (encodedString[i + j] - '0' + 52);
+
+            // '+' occurs in 62nd position in charSet
+            else if (encodedString[i + j] == '+')
+                bitstream = bitstream | 62;
+
+            // '/' occurs in 63rd position in charSet
+            else if (encodedString[i + j] == '/')
+                bitstream = bitstream | 63;
+
+            else {
+                bitstream = bitstream >> 2;
+                countBits -= 2;
+            }
+        }
+
+        while (countBits != 0) {
+            countBits -= 8;
+            decodedString[k++] = (bitstream >> countBits) & 255;
+        }
+    }
+}
+
+static EB_ERRORTYPE ParseSeiMetaData(
+    SequenceControlSet_t        *sequenceControlSetPtr,
```

Comment:
Please do not parse data from file in the library. You could first do it in the application, then copy to library.